src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {

```

Comment:
According to the checklist, we should follow the encapsulation principle. The `fruitRepository` should not be public because it exposes internal storage details. Consider making it private and providing access through methods.